File: Python/SQLAlchemy/src/models/department.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship

from .base import Base


class Department(Base):
    __tablename__ = 'departments'

    id = Column(Integer, primary_key=True)
    name = Column(String)
    employees = relationship(
        'Employee',
        back_populates='department',
        cascade="all, delete"
    )

    def __str__(self):
        return (
            "Department { "
            f"Id: {self.id}, "
            f"Name: {self.name} "
            "}"
        )

File: TypeScript/MicroORM/models/Department.ts
import { Entity, Property, PrimaryKey } from '@mikro-orm/core';

@Entity({ tableName: 'departments' })
export class Department {
    @PrimaryKey()
    public id!: number;

    @Property()
    public name!: string;

    string(): string {
        return `{ Department { Id: ${this.id}, Name: ${this.name} } }`;
    }
}
